repos/system_upgrade/el8toel9/actors/cloud/checkvalidgrubcfghybrid/libraries/checkvalidgrubcfghybrid.py

```diff
@@ -0,0 +1,27 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import HybridImage
+
+
+def process():
+    hybrid_image = next(api.consume(HybridImage), None)
+
+    if hybrid_image:
+        reporting.create_report([
+            reporting.Title(
+                'Azure hybrid (BIOS/EFI) image detected. The GRUB configuration might be regenerated.'
+            ),
+            reporting.Summary(
+                'Leapp detected the system is running on Azure cloud, booted using BIOS In case '
+                'of such a hybrid image scenario and upgrading from older systems (i.e. RHEL 7) '
```

Comment:
These sentences is hard for me to understand, likely because I am old :older_man: